tests/snapshots/generate_snapshots.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+import itertools
+import subprocess
+import pathlib
+
+def generate_snapshots():
+    single_styles = ["changes", "grid", "header", "numbers"]
+    collective_styles = ["full", "plain"]
+
+    for num in range(len(single_styles)):
+        for grouped in itertools.combinations(single_styles, num + 1):
+            generate_snapshot("--style={}".format(",".join(grouped)))
+
+    for style in collective_styles:
+        generate_snapshot("--style={}".format(style))
+
+    generate_snapshot("--color=always")
+    generate_snapshot("--list-languages")
```

Comment:
I'm not sure if `--list-languages` and `--list-themes` should be added as a snapshot test. We would have to re-generate those files every time we add a new syntax or theme.

I'm also not quite sure about the `--color=always` test because it also depends on the Default highlighting theme.

Reply:
Alright, I wasn't sure about those either. I added them after the fact.